src/fancyrag/db/neo4j_queries.py

```
@@ -0,0 +1,346 @@
+        try:
+            value = record[0]  # type: ignore[index]
+        except Exception:  # pragma: no cover - defensive guard
+            value = None
+    return int(value or 0)
+
+
+def _normalise_records(result: ValueResult) -> Sequence[Mapping[str, Any]]:
+    if isinstance(result, tuple):
+        result = result[0]
+    if hasattr(result, "records"):
+        return getattr(result, "records")
```

Ruff (0.13.3)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)